### PR TITLE
Make sure there is less objects created

### PR DESCRIPTION
**Pull request by Nyholm**
This is just a minor thing.

### CODE REVIEW

File: src/ResponseFactory.php [hunk 1]
```diff
@@ -9,7 +9,6 @@ class ResponseFactory implements ResponseFactoryInterface
 {
     public function createResponse($code = 200)
     {
-        return (new Response())
-            ->withStatus($code);
+        return new Response('php://memory', $code);
```
Comment (shadowhand) [inline src/ResponseFactory.php]: Having to define `$body` is why I didn't write it this way. What's the intended benefit of this change?

Comment (Nyholm) [inline src/ResponseFactory.php]: Just a super minor. If we use the default value for this arguments then there is one less object that will be instantiated.

Comment (shadowhand) [inline src/ResponseFactory.php]: And what if the default changes to something undesirable in the future? Having one less object isn't worth it in my opinion.

Comment (Nyholm) [inline src/ResponseFactory.php]: If their default changes to something undesirable in the future we will be happy we wrote `'php://memory'` here. 

> Having one less object isn't worth it in my opinion.

That is fair. Let's close this PR then.

**Pull request closed without merging**